daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2698,11 +2726,55 @@ func NewDeviceSet(root string, doInit bool, options []string, uidMaps, gidMaps [
 				return nil, err
 			}
 			devices.xfsNospaceRetries = val
+		case "dm.directlvm_device":
+			lvmSetupConfig.Device = val
```

Comment:
If there is a filesystem on the device, `pvcreate` asks if it should be wiped. Since there's no stdin for the process it causes docker to error out, with the output from pvcreate in the logs.
This is a pretty bad experience, though at least it won't overwrite someone's existing stuff automatically.

We can check 2 things:
1. Does this device exists
2. Is there a filesystem on the device

For check 2, we can we can add an option to allow forcing the overwrite but I do not think we should just overwrite by default.